p2p/stdnetwork/network.go

```diff
@@ -20,21 +20,21 @@ var (
 type network struct {
 	addrBookMap typesP2P.AddrBookMap
 
-	logger modules.Logger
+	logger zerolog.Logger
```

Comment:
We have our `modules.Logger` abstraction so we don't need to expose `zerolog` anywhere outside of `logger`. I can't find `logger zerolog.Logger` anywhere else in the code. 

Can we avoid this

Reply:
We noticed that and we were confused by the fact that VSCode showed this: 

![image](https://user-images.githubusercontent.com/29378614/217894264-9e158679-c128-4b7f-95cc-30f8747aa53b.png)

Already fixed :)